src/app/space/app-launcher/create-app/create-app.component.ts

```diff
@@ -79,6 +81,11 @@ export class CreateAppComponent implements OnDestroy, OnInit {
           .pipe(map(workSpaceLinks => {
             window.open(workSpaceLinks.links.open, '_blank');
           }));
+      } else {
+        this.notifications.message({
+          message: `Che cluster is full`,
```

Comment:
Knowing the che cluster is full is part of the message we want to convey. A good error message should contain 3 parts: context, the error and an action the user can take.

Context: Unable to create workspace.
Error: Che cluster is full.
Action: How can the user resolve this? Can they resolve it? What action can they take? Contact admin? Is there something they can do themselves?

Reply:
I am not sure. What to do when Che cluster is full? 

@joshuawilson Do you have any idea about this? What user will do when Che cluster is full?

Reply:
I'm not overly concerned with the error message at this time as I'm sure we have poor messages scattered across the app that make a sweep worth while. But it would be great if we could provide the user with some guidance.

Once everything else is addressed, if this is the last outstanding issue, make a new work item and we can move on this pr.

Reply:
@ibuziuk are you able to tell us what we can do or know when che cluster if full?